Assets/HoloToolkit-Examples/Input/Scripts/MicrophoneHelperSample.cs

```diff
@@ -12,14 +12,17 @@ public class MicrophoneHelperSample : MonoBehaviour
 #if ENABLE_WINMD_SUPPORT
         private async void Update()
 #else
+        //Update is called one per frame
         private void Update()
 #endif
         {
+            //If button M is pressed
```

Comment:
What if the dev decided to use spacebar?

Reply:
Sorry that I just followed the declaration of the variables earlier in the code, I'll try my best to modify it again. Thanks a lot!